lms/djangoapps/courseware/views.py

```diff
@@ -985,7 +992,7 @@ def _progress(request, course_key, student_id):
         'courseware_summary': courseware_summary,
         'studio_url': studio_url,
         'grade_summary': grade_summary,
-        'staff_access': staff_access,
+        'staff_access': has_access_on_students_profiles,
```

Comment:
This is misleading -- you're changing the meaning of `staff_access` in the templates. Can you rename this variable in the template context, and update the template to use the renamed variable?